### ISSUE TITLE
Instability when looking for bones

### ISSUE THREAD

**Issue by ekdrnqkdrn** [microsoft/Azure-Kinect-Sensor-SDK#1751]
Describe the bug
The skeleton was not clearly visible when wearing the mask. 
And there is a phenomenon that the skeleton does not hang at all when lying down.
Could you please tell me how I can solve this problem?
I will be very grateful.

To Reproduce
<!-- Steps to reproduce the behavior:
1. Launch 'k4abt_simple_3d_viewer.exe'
2. A black mask is more prone to problems than a white mask.

Expected behavior
I expect that the body tracking algorithm behave the same no matter what kind of adult person is in front of the camera.

Logs

Screensho
![screenshot1](https://user-images.githubusercontent.com/75236024/156122598-9f0722cd-7cd5-42be-b549-c9a95a21c129.png)
![scrennshot2](https://user-images.githubusercontent.com/75236024/156122601-4e4d8820-e4b8-4aff-8791-ce2d88ea1f6f.png)
ts

Desktop (please complete the following information):
 - OS with Version: Windows 10 Pro
 - SDK Version: Camera 1.4.1, Body Tracking 1.1.0
 - Firmware version: RGB Camera: 
![image](https://user-images.githubusercontent.com/75236024/156120682-cc142ff5-bb85-4d45-a6d6-65a04f0e9c13.png)
![version](https://user-images.githubusercontent.com/75236024/156122615-0149d04a-8e1f-4c9d-921d-bb3096d066d4.png)

 - GPU: NVidia GeForce RTX 2070 SUPER

Additional context

**Comment by ekdrnqkdrn** [comment]
![image](https://user-images.githubusercontent.com/75236024/156122075-2b542a87-71f4-41f0-9c05-a16c4ed72872.png)

**Comment by ekdrnqkdrn** [comment]
The picture didn't upload, so I'll post it here.

**Comment by ekdrnqkdrn** [comment]
![image](https://user-images.githubusercontent.com/75236024/156122220-0bb8d2b5-9a4d-40cb-be46-b51a7faaa9de.png)

**Comment by ekdrnqkdrn** [comment]
![image](https://user-images.githubusercontent.com/75236024/156122262-a0b81427-ea0b-4a99-bc1b-67083fb1c285.png)

**Comment by Rudizzzz** [comment]
This is related to my problem #1634  posted 8 months ago..

**Comment by owen2t** [comment]
In my experience, the body tracking works best when wearing tight fitting clothes, perhaps that would help. 

I have also noticed that the body tracking performs poorly when the person is lying down/doing a pushup. My hypothesis about why this is happening is that the skeleton is estimated using a neural network, which means it is only as good as its training data- and I don't think there are a lot of examples of a person lying down in the training data, so the body tracking will usually perform poorly. But that's just my hypothesis so don't take it as fact!

**Comment by fkavache** [comment]
Hi @ekdrnqkdrn

I'm wearing a facemask too, maybe it depends on mask material, in your case BT is unable to detect depth on the mask area.

![image](https://user-images.githubusercontent.com/29062515/163359111-9e5a6d2e-f562-497c-bb67-6283cc5ae507.png)

**Comment by ekdrnqkdrn** [comment]
Hi @fkavache 
I noticed that the recognition rate dropped significantly when I wore a black mask.
Is there any way to increase the recognition rate?
